WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/details/BackupDownloadDetailsFragment.kt

```diff
@@ -0,0 +1,59 @@
+package org.wordpress.android.ui.jetpack.backup.details
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.ui.jetpack.backup.BackupDownloadViewModel
+import javax.inject.Inject
+
+class BackupDownloadDetailsFragment : Fragment() {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var parentViewModel: BackupDownloadViewModel
+    private lateinit var viewModel: BackupDownloadDetailsViewModel
```

Comment:
Minor (🔍 ): Consider adding an empty line to separate the last field from the first method.

PS: The same applies to both `ViewModel` classes, having the `isStarted` separated by an empty line with the `start()` function.